### ISSUE TITLE
Add "Community and Help" section

### ISSUE THREAD

**Issue by nponeccop** [Gabriel439/post-rfc#131]
- How do I find a library?
- How do I find a solution?
- How do I find the most idiomatic way to solve this problem in Haskell?
- There are numerous blogs with new ideas (Comonad Reader etc)
- There are many user groups in different languages in IRC, Gitter, Telegram, Facebook - you name it.
- StackOverflow has 3 useful sites for a Haskeller: StackOverflow, CodeReview and ComputerScience (and what is more important is that if you ask a harder question which only Edward Kmett or Don Stewart can answer they probably will there, up to the point of https://ro-che.info/ccc/2)

**Comment by Gabriella439** [comment]
Yeah, I think this is a good idea.  I think another useful thing would be some rough rules of thumb for evaluating library quality (i.e. number of downloads, documentation quality)

**Comment by ad-si** [comment]
This section should probably also include Documentation (Haddock etc.), or do you think a separate section would be better?

**Comment by nponeccop** [comment]
I agree
